lib/ret_web/router.ex

```diff
@@ -127,9 +128,9 @@ defmodule RetWeb.Router do
       resources("/slack", Api.V1.SlackController, only: [:create])
     end
 
-    scope "/v1/internal", as: :api_v1 do
+    scope "/api-internal/v1", as: :api_v1 do
```

Comment:
Is there a reason we want these routes on both `/api/api-internal/v1` and `/api-internal/v1`?